packages/ui/components/ui/dutch-auction-component/progress-bar/indicator/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { DutchAuction } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/auction/v1/auction_pb';
+import { CircleArrowRight, CircleCheck, CircleX } from 'lucide-react';
+import { getProgress } from './get-progress';
+import { getDescription } from '@penumbra-zone/getters/dutch-auction';
+import { isZero } from '@penumbra-zone/types/amount';
+
+export const Indicator = ({
```

Comment:
Moved from one folder up, but there were enough changes that git thinks it's a whole new file. Basically I just pulled more logic into `<Indicator />`.